manifest.json

```diff
@@ -4330,6 +4330,28 @@
             }
           ]
         },
+        {
+          "path": "specifications/ewd998/EWD998PCal.tla",
+          "communityDependencies": [
+            "BagsExt"
+          ],
+          "tlaLanguageVersion": 2,
+          "features": ["pluscal"],
+          "models": [
+            {
+              "path": "specifications/ewd998/EWD998PCal.cfg",
+              "runtime": "00:01:00",
```

Comment:
If the runtime is one minute this spec doesn't count as a small spec; those are for specs that execute in less than 10 seconds.